Documentation/ApiOverview/BackendModules/BackendModuleApi/Index.rst

```diff
@@ -117,9 +117,6 @@ be used to add submodules to this new toplevel module:
 .. note::
    The main module name should contain only lowercase characters. Do not use an underscore or dash.
 
-.. index:: $GLOBALS; TBE_MODULES
```

Comment:
Why are you removing the index and link target here?

Reply:
It was a duplicate and i removed it from this page as this page is not indexed in any menu.

Reply:
The page needs to be marked as an orphan if it is not in the toctree. And I think changes like these are beyond the scope of this PR.

Reply:
Yes, so, this should have been done by the creator of the page. I did not remove it from the menu, just used this as argument to remove the duplicate index on this page instead of the other.